walkthrough/customizingAppearancefr.md

```diff
@@ -0,0 +1,20 @@
+## Personnalisation de l'apparence
```

Comment:
The files in the `walkthrough` folder should end with `.nls.fr.md` . This one should be `customizingAppearance.nls.fr.md` instead.

Update the filename of the other 5 files too.